egs2/TEMPLATE/diar_enh1/pyscripts/utils/convert_rttm.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
I saw there are some files that are duplicated from the diar codebase, maybe we can try to use softlink for that if no changes applied?

Reply:
Thanks for the review! I replaced them with softlinks.